src/client/error.rs

```diff
@@ -52,13 +52,13 @@ pub enum ClientError {
     ParcelAlreadyExists,
     /// The error returned when the request is invalid. Contains the underlying HTTP status code and
     /// any message returned from the API
-    #[error("Invalid request (status code {status_code:?}): {message:?}")]
+    #[error("Invalid request (status code {status_code:?}): {}", .message.clone().unwrap_or_else(|| "unknown error".to_owned()))]
     InvalidRequest {
         status_code: reqwest::StatusCode,
         message: Option<String>,
     },
     /// A server error was encountered. Contains an optional message from the server
-    #[error("Server has encountered an error: {0:?}")]
+    #[error("Server has encountered an error: {}", .0.clone().unwrap_or_else(||"Protocol error. Verify the Bindle URL".to_owned()))]
```

Comment:
From what you said this could result from the wrong port or from a network disconnect?  If so I feel like "Server has encountered an error" may send people down the wrong path here... but also very hard to get clear wording for what could be a wide range of cases that are hard to tell apart!

Reply:
Maybe "Networking error: {}" would be more accurate

Reply:
Ah! "Error contacting server"